less/opendev.less

```diff
@@ -12,6 +12,7 @@ img {
 }
 
 table {
```

Comment:
I test a topic page which contains the table, set the width to 80% will show ugly.
https://opendevelopmentcambodia.net/topics/protected-forest/
![screenshot_9](https://user-images.githubusercontent.com/6613530/28620199-209d0948-7236-11e7-9198-1c2568804877.jpg)

Please ensure that all the tables are using on the topic pages will look fine.